pvlib/solarposition.py

```diff
@@ -1210,3 +1210,50 @@ def hour_angle(times, longitude, equation_of_time):
     )).total_seconds() / 3600. for t in times])
     timezone = times.tz.utcoffset(times).total_seconds() / 3600.
     return 15. * (hours - 12. - timezone) + longitude + equation_of_time / 4.
+
+
+def _hours(times, hour_angle, longitude, equation_of_time):
+    timezone = times.tz.utcoffset(times).total_seconds() / 3600.
```

Comment:
hmm odd to see `times` as both the object and explicit argument of its own method.

Reply:
it is weird, but I think it's the only way, I broke it up into two lines that explains a bit more:
```python
tz_info = times.tz  # this is the pytz timezone info object that has the utcoffset method
tz = tz_info.utcoffset(times)  # this method requires a datetime-like argument to operate on
```